browser/extensions/brave_component_loader.h

```diff
@@ -41,6 +41,7 @@ class BraveComponentLoader : public ComponentLoader {
 #if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
   void AddEthereumRemoteClientExtension();
   void AddEthereumRemoteClientExtensionOnStartup();
+  void RemoveEthereumRemoteClientExtension();
```

Comment:
Please rename all of these `UnloadEthereumRemoteClientExtension` because we aren't actually removing the ERC extension here.

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558